communities/asu-phoenix/community-impact-proposal.md

```diff
@@ -0,0 +1,123 @@
+
+- **Community demographics**:
+    - 5,560 students just in Computing Program Alone 
+    - Want to reach out to different majors
+        - Digital Culture (Arts)
+        - Technical Management (Business)
+        - Graphic Information Technology (Design)
+        - Human Systems Engineering (HCI)
+        - Computing Information Systems (Business)
+    - Want to reach out to the exisiting clubs and tech influences that was formally recruited for this past hackathon 
+    #### Exisiting organizations and Tech Influences
+- ASU, Large University, Multiple Organizations
+    - Computer Science Organizations 
+        - Mobile Devs
+        - Video Game Developer Club
+        - Software Developer Association 
+        - Women in Computer Science 
+        - Technology + Innovation Lab 
+        - Sun Devil Robotics
+        - Intern Supply 
+        - PWN Devils 
+        - CodePath
+        - Desert Hacks
+        - Video Game Development 
+    - Digital Culture Organizations 
+        - Digital Culture Underground 
+        - [Other](https://artsmediaengineering.asu.edu/resources/students/clubs-organizations) 
+    - Engineering Organizations 
+        - Fulton Ambassadors 
+        - Fulton Student Council 
+        - Polytechnic Campus Orgs
+        - [Fulton Student Orgs](https://fso.engineering.asu.edu/student-org-directory/)
+
+- Phoenix Area, many partnerships + meetups 
+    - Galvanize, great co-working/event space
+    - #YesPHX Movement (Tech Startup Movement)
+    - Meetups 
+        - Code For America
+        - VR/AR Hangouts 
+        - IoT Meetups 
+    - Hackerrnest socials 
+    - Local Companies 
+        - GoDaddy 
+        - Intel 
+        - Amazon
+        - American Express 
+
+- **Culture**: There is a new excitement in the air, the tech scene is growing fast in the Phoenix Area and more people are getting incolced some way or the other even if not directly CS. 
+
+
+## Your Proposal
+
+Create a unifying way to showcase the existing organizations and tech influences to a community not only of veterns but the freshmen especially. This would involve hosting an event right after coming back from school. Currently there is a passport day that is successful to showcase ALL different clubs. There could possibly be a specific Tech centered one that really allows all those interested in Tech to see all the different opportunities of involvement they themselves can have in the ASU/Phoenix community. 
+
+
+- **Timing**: 
+
+| Week |Title|Description| 
+|:----:|:-------------------------------:|:-------------------------------------------------------------------------------------------------------------------------------------------------:|
+|   3  | Research Past Events/Choose Event Type | Understand which freshmen/beginner friendly events work best. Large Scale Hackathon? University Wide Tech Day? 
```

Comment:
A university wide tech day would align really well with some of the groups you want to incorporate above from other majors! You can have a small hackathon as part of it to include your core membership